src/woly/atoms/box/index.ts

```diff
@@ -47,3 +47,11 @@ export const boxVertical = css`
     padding: 0;
   }
 `;
+
+export const BoxComponent = styled.div`
```

Comment:
Лучше перенести в atoms

Reply:
```suggestion
export const Box = styled.div`
```

Я бы не стал называть его так длинно

Reply:
Box уже находится в директории атом